mlx_engine/generate.py

```
@@ -104,6 +120,7 @@ def create_generator(
     seed: Optional[int] = None,
     json_schema: Optional[str] = None,
     max_tokens: Optional[int] = 10000000,
+    num_draft_tokens: Optional[int] = None,
```

Question: should this have a default value?

---

Didn't default it to make it possible that we can say "hey this isn't gonna work" if you try to set num_draft_tokens and there is no draft model loaded. The other option is to silently ignore it, I felt like being explicit could be better but I don't feel strongly if we want to just have this default but its just never used if no draft model